tests/modules/services/window-managers/i3/i3-null-config.nix

```
@@ -0,0 +1,18 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    xsession.windowManager.i3 = {
+      enable = true;
+      config = null;
+    };
+
```

Should add something like
```suggestion
    nixpkgs.overlays = [
      (self: super: {
        dmenu = super.dmenu // { outPath = "@dmenu@"; };
        i3 = super.writeScriptBin "i3" "" // { outPath = "@i3@"; };
        i3-gaps = super.writeScriptBin "i3" "" // { outPath = "@i3-gaps@"; };
        i3status = super.i3status // { outPath = "@i3status@"; };
      })
    ];

```
to avoid unnecessary downloads in the CI job.